src/bin/outpack/args.rs

```diff
@@ -0,0 +1,35 @@
+#[derive(clap::Parser, Debug)]
+pub struct Args {
+    #[command(subcommand)]
+    pub command: Command,
+}
+
+#[derive(clap::Subcommand, Debug)]
+pub enum Command {
+    /// Initialize a new outpack repository
+    Init {
+        path: String,
+
+        #[arg(long)]
+        path_archive: Option<String>,
+
+        #[arg(long)]
+        use_file_store: bool,
+
+        #[arg(long)]
+        require_complete_tree: bool,
+    },
+    /// Search for a packet in a repository
+    Search {
+        #[arg(short, long)]
+        root: String,
+        query: String,
+    },
+    /// Parse an outpack query, without evaluating it
+    Parse { query: String },
+    /// Start the outpack API server
+    ApiServer {
```

Comment:
How about just `Start`?

Reply:
The command is called `outpack`, so `outpack start` seems pretty ambiguous IMO.
Alternatively we could keep the old `outpack_server` command name, but then it makes the other commands a bit odd.

Reply:
Gone with `StartServer`.